exec/cnex/lib/io.c

```diff
@@ -38,7 +64,7 @@ static FILE *check_file(TExecutor *exec, void *pf)
 void io_fprint(TExecutor *exec)
```

Comment:
`io.fprint` no longer exists, it's `io.writeLine` now.

Reply:
To be clear, did you mean `textio.writeLine()`?  There is no exported function writeLine in io.neon/cpp.  I will remove this function from io.c.

Reply:
Yes, sorry I did mean `textio.writeLine`. Also, there is no `io.readLine`. That's the whole reason for existence of the `textio` module.

Reply:
Removed io.readLine() also.